source/horoscope/css/landing.css

```diff
@@ -141,41 +141,53 @@ input[type="radio"]:checked + label {
     font-size: 1.5em;
 }
 
-nav {
-    position: fixed;
-    bottom: 20px;
-    right: 20px;
-    display: flex;
-    flex-direction: column;
-}
-  
-#history-button {
-    background: none;
-    border: none;
+.navbar {
+    list-style-type: none;
     padding: 0;
-    cursor: pointer;
+    margin: 0;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    border: 1px solid #ccc;
+    z-index: 1000;
```

Comment:
do we need such high of a z-index? Do we even need it? Doesn't look different without the rule